src/main/java/webserver/data/Todos.java

```diff
@@ -0,0 +1,14 @@
+package webserver.data;
+
+import java.util.List;
+import java.util.Map;
+
+import webserver.models.Todo;
+
+interface Todos {
+    public List<Todo> getTodos();
+
+    public void addTodo(Map<String, String> body, int id);
```

Comment:
This might be _too_ generic.  You might keep this method (since you're already using it and is working) but add another one that's more specific

```java
public void addTodo(int id, String heading, String bodyText, boolean isDone);
```

Reply:
I really like how you can have the same method name but based off the parameters, it can be more flexible.